### PR TITLE
chore: added appInsights alerts for quartz jobs (FPLA-2801)

### PR DESCRIPTION
**Pull request by Jamesnelson117**
**Before creating a pull request make sure that:**

- [x] PR's title includes JIRA ticket number and follow [Conventional Commits](https://www.conventionalcommits.org/) specification
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-2801

### Change description ###
Added appInsights alerts for 'summary tab' and 'upcoming hearings' quartz job.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: service/src/main/resources/application.yaml [hunk 1]
```diff
@@ -146,7 +146,7 @@ scheduler:
       cronGroup: "NIGHTLY_CRON"
       description: 'Finds upcoming hearings'
       jobClass: 'uk.gov.hmcts.reform.fpl.jobs.UpcomingHearingsFinder'
-      cron: ${UPCOMING_HEARINGS_CRON:0 0 2 ? * MON-FRI *}
+      cron: ${UPCOMING_HEARINGS_CRON:0 0 2 ? * * *}
```
Comment (Jamesnelson117) [inline service/src/main/resources/application.yaml:149]: Not needed as job runs `!calendarService.isWorkingDay(baseDate)` to determine wether to run or not.

Comment (tomaszpowroznik) [reply service/src/main/resources/application.yaml:149]: should be ok 👍

**Comment by tomaszpowroznik** [comment]
following errors occurs:
```
error loading module [.]: Invalid character: Single quotes are not valid. Use double quotes (") to enclose strings. (and 2 other messages)

Error loading infra definition at .

**Review by lewisbirks — APPROVED** [review]
@samuelzerozero can you have a look over this just to double check it as I am not familiar with it

**Pull request merged**